interbtc-spec/docs/source/spec/fee.rst

```diff
@@ -95,89 +78,79 @@ Fee for users to premium redeem (as percentage). If users execute a redeem with
 - Paid in collateral
 - Initial value: 5%
 
-.. _PunishmentFee:
+.. _punishmentFee:
 
 PunishmentFee
 .............
 
-Fee (as percentage) that a vault has to pay if it fails to execute redeem requests (for redeem, on top of the slashed value of the request). The fee is paid in collateral based on the wrapped token amount at the current exchange rate.
+Fee (as percentage) that a Vault has to pay if it fails to execute redeem requests (for redeem, on top of the slashed value of the request).
+The fee is paid in collateral based on the ``interBTC`` amount at the current exchange rate.
 
 - Paid in collateral
 - Initial value: 10%
 
-PunishmentDelay
-...............
-
-Time period in which a vault cannot participate in issue, redeem or replace requests.
-
-- Measured in Parachain blocks
-- Initial value: 1 day (Parachain constant)
-
-.. _ReplaceGriefingCollateral:
+.. _replaceGriefingCollateral:
 
 ReplaceGriefingCollateral
 .........................
 
-Default griefing collateral as a percentage of the to-be-locked collateral of the new vault, vault has to lock to be replaced by another vault. This collateral will be slashed and allocated to the replacing Vault if the to-be-replaced Vault does not transfer BTC on time.
+Default griefing collateral as a percentage of the to-be-locked collateral of the new Vault, Vault has to lock to be replaced by another Vault.
+This collateral will be slashed and allocated to the replacing Vault if the to-be-replaced Vault does not transfer BTC on time.
 
 - Paid in collateral
 - Initial value: 0.005%
 
-
 Functions
 ~~~~~~~~~
 
 distributeRewards
 -----------------
 
-Specifies the distribution of fees among incentivised network participants.
-
+Distributes fees among incentivised network participants.
 
 Specification
 .............
 
 *Function Signature*
 
-``distributeRewards()``
+``distributeRewards(amount)``
 
+*Preconditions*
```

Comment:
Looks like pre-conditions are missing here. For this to pass, I assume we have to have at least 1 vault registered and active?